lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -202,7 +203,7 @@ class window.Turbolinks
       triggerEvent 'page:change'
       triggerEvent 'page:update'
 
-    return
+    return [].slice.call(body.children)
```

Comment:
You're right, step 1 seems silly in that case. 

I'd like to see a refactor for sure.

The return value was changed because it was strange that it returned nothing when you changed all the nodes, I think. Memory is fuzzy after vacation. If that's the case I should have moved the change to another PR. I think maybe I was writing tests for the promise behaviour and ran into issues with inspecting the outcome from TG's apis in tests that dealt with full body changes?

Anyway, I'm cool with moving it to another PR if we want so we can talk about it in more detail, or dropping it.